.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Build
+
+on:
+  push:
+
+permissions: {}
+
+jobs:
+  linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+      - name: Install GCC 9
+        run: |
+          sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+          sudo apt update
+          sudo apt install g++-9
+      - name: Install SDL2
+        run: |
+          curl https://www.libsdl.org/release/SDL2-2.0.10.tar.gz | tar xzf -
+          mkdir SDL2-2.0.10/build
+          cd SDL2-2.0.10/build
+          ../configure && make -j2 && sudo make install
```

Comment:
On Ubuntu, we could save a lot of time by installing `libsdl2-dev` instead (the default release of Ubuntu on GHAs, 20.04 currently, has SDL 2.0.10); but that would tie the build to whatever is the default on whatever release of Ubuntu we get at a given point in time.

Reply:
That would be fine.  As I generally use Ubuntu for development anyway, there is no issue with tying it.  Please add this if you like.

Reply:
Done!